ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -187,3 +187,24 @@ def _rad2deg(
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
+
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def _degrees(
+    x,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="K",
+    dtype=None,
+    subok=True,
+):
+    ret = ivy.degrees(x, out=out)
```

Comment:
the function you're looking for should be the name `rad2deg` in Ivy, please then ensure that the used arguments are matching the function signatures

Reply:
no i was adding degrees function of numpy

Reply:
what I meant was `rad2deg` is the equivalent `degrees` in Ivy, basically does the same thing

`ivy.degrees` can't be used here because it doesn't exist and will throw an error, the numpy frontend function name should still be `degrees` but the ivy api used should be `rad2deg`